drivers/lora/sx1276.c

```diff
@@ -126,7 +132,9 @@ void SX1276SetAntSwLowPower(bool status)
 
 void SX1276SetBoardTcxo(u8_t state)
 {
-	/* TODO */
+#if DT_INST_NODE_HAS_PROP(0, tcxo_power_gpios)
+	gpio_pin_set(dev_data.tcxo_power, GPIO_TCXO_POWER_PIN, state ? 1 : 0);
```

Comment:
It is actually a good idea to include something like `tcxo-power-startup-delay` DT property.

Reply:
What I can see is that we do not provide `GetWakeupTime` callback as well. So it would be a good idea to start doing it, together with the introduction of above DT property.

Reply:
I noticed that as well after posting the comments. Another think I noticed (I started implementing pretty much the same changeset before finding you pull request) was that the reference LoRaWAN implementation turns on the TCXO in from the reset handler. I don’t know if that makes sense, but it’s probably a good idea to program a known state both in the TCXO and the antenna switch.

Reply:
I think that TCXO and antenna switch have known state already, because all pins are initialized with GPIO_OUTPUT_INACTIVE. However I agree with `SX1276Reset` case, we should do the same as reference implementation (power on TCXO before asserting reset pin).

Reply:
> > Any chance you could add support for a 3-way switch that selects between RX/TX_RFO/TX_PA? That seems to be what the Murata ABZ-078 uses. The reference implementation checks if PA boost has been enabled when enabling the TX path and selects the appropriate TX output in that case. This would likely have knock-on effect on antenna selection at some point.
> 
> So far sx1276 driver allows to select between RFO and PA_BOOST output in build time only. With that restriction in mind we do not need to have 3-way switch - 2-way switch is enough (either RX/TX_RFO or RX/TX_PA_BOOST).
> 
> If we would like to select between RFO and PA_BOOST in runtime, then we need to rework functionality selected by `power-amplifier-output` (or `CONFIG_PA_RFO_PIN` in master branch). @andysan Are there actually some use cases when you want to select between RFO and PA_BOOST in runtime? Have you some in mind?

My understanding is RFO path is significantly more power efficient than the PA boost path for the output powers where they overlap. I.e., if you are implementing a LoRa-WAN device that dynamically adjusts the output power (I believe this is pretty common), you'd get much better battery life if you disable PA boost when it isn't needed. Common platforms like the B-L072Z (Murata-based) hae a 3-way switch. The reference firmware only enables PA boost when the output power is higher than 14dB.